client/my-sites/site-settings/site-owner-transfer/site-owner-transfer.tsx

```diff
@@ -6,15 +6,26 @@ import FormattedHeader from 'calypso/components/formatted-header';
 import HeaderCake from 'calypso/components/header-cake';
 import Main from 'calypso/components/main';
 import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import { ResponseDomain } from 'calypso/lib/domains/types';
 import { TRANSFER_SITE } from 'calypso/lib/url/support';
+import { getDomainsBySiteId } from 'calypso/state/sites/domains/selectors';
 import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import PendingDomainTransfer from './pending-domain-transfer';
 import StartSiteOwnerTransfer from './start-site-owner-transfer';
 
 const SiteOwnerTransfer = () => {
 	const selectedSiteId = useSelector( ( state ) => getSelectedSiteId( state ) );
```

Comment:
What do you think about using the `getSelectedSite` selector instead? It looks like better typed so that we wouldn't need the `as unknown as number ` casting later in the flow.